packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -526,6 +527,18 @@ const getEchartAxis = ({
                     return formatItemValue(field, value, true);
                 },
             };
+        } else if (
+            isDimension(field) &&
+            (field.timeInterval === TimeFrames.WEEK ||
+                field.timeInterval === TimeFrames.DAY)
+        ) {
+            // If timeInterval is WEEK or DAY, we don't show time on the axis
+            // We don't use axisMinInterval becuase that breaks the dynamic format on dates
+            axisConfig.axisLabel = {
+                formatter: {
+                    hour: '',
+                },
```

Comment:
Didn't know about this echarts setting 😮  really nice 🕺 
What do you think if we had this configuration in the `timeFrameConfigs` as we have for `axisMinInterval`?

Reply:
yeah, good idea, if we are happy about formatter I can make the change [](edited-with-Axolo)

Reply:
I’m happy with the formatter :+1: [](edited-with-Axolo)